pandas/core/groupby/generic.py

```diff
@@ -583,7 +583,8 @@ def transform(self, func, *args, **kwargs):
             if not (func in base.transform_kernel_whitelist):
                 msg = "'{func}' is not a valid function name for transform(name)"
                 raise ValueError(msg.format(func=func))
-            if func in base.cythonized_kernels:
+            # transformation are added as well since they are broadcasted already
+            if func in base.cythonized_kernels or func in base.transformation_kernels:
                 # cythonized transformation or canned "reduction+broadcast"
```

Comment:
the added comment should either a) go after the condition and indented, or have a newline before the comment

Reply:
added a newline!